.gitignore

```diff
@@ -0,0 +1,23 @@
+# User-specific files
+*.suo
+*.user
+
+# Build results
+[Bb]in/
+[Oo]bj/
+*.csproj.user
+*.pubxml.user
+
+A-vs-An/PerfTest/354984si.ngl
+
+*.vsp
+*.psess
+*.suo
+*.vspx
+packages/
+*.DotSettings.user
+*.nupkg
+*.received.txt
+*.approved.txt.bak
```

Comment:
I copied the .hgignore and made some changes changes to the format to work for me. I typically use https://www.gitignore.io/api/visualstudio in my own projects, it's pretty massive though and has a lot of redundant lines, but makes life simple.